frontend/src/App.js

```diff
@@ -180,21 +184,87 @@ class App extends Component {
         const barSize = 8;
         const recoSize = 4;
 
-        let headline = (this.state.hasSearched ? "My food impact" : "Track the climate impact of my food")
+        let headline = (this.state.hasSearched ? "\u2014 My food impact \u2014" : "Track the climate impact of my food")
+
+        // Override global themes for Typography. TODO: place in separate imported doc. like index.css
+        const theme = createMuiTheme({
+            typography: {
```

Comment:
I made a small change here to the order and committed to your branch

Reply:
Awesome. Thanks for looking into it.